src/Filter.js

```diff
@@ -10,20 +10,143 @@ function Filter({ dataset, schema, filter, setFilter }) {
   // getTotalRows.filter({where: {ConnectedArea: {_eq: 'DK1'}}});
   const QUERY = gql`
     query Dataset {
-      ${getTotalRows}
+      ${getTotalRows} 
```

Comment:
it's very minor but we don't need extra whitespace here 😄

Reply:
ok